moon/controller.py

```diff
@@ -78,6 +78,7 @@ def __init__(self, config, bus):
         super().__init__(config, bus)
         bus.register("desired_speed", "artf_xyz", "artf_cmd", "pose2d", "pose3d", "driving_recovery", "request", "cmd")
 
+        self.sim_time = None
```

Comment:
It may be a valid way, the reasons I didn't pursue it are 1) we may want to have both real time and sim time 2) wasn't sure of the implications throughout OSGAR by changing the meaning of self.time